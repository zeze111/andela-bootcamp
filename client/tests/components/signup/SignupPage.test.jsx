import React from 'react';

import SignupPage from '../../../components/signup/Signuppage';
import store from '../../../store';

const props = {
  signUp: jest.fn(() => Promise.resolve()),
  signIn: jest.fn(() => Promise.resolve())
}

const setup = () => {
  const wrapper = shallow(<SignupPage {...props} store={store} />);
  return { wrapper }
}

describe('Test for Sign up page', () => {
  it('should render component', () => {

    const wrapper = setup();
    expect(wrapper).toMatchSnapshot()
  });
});
