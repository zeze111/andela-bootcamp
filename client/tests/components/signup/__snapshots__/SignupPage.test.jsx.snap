// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Sign up page should render component 1`] = `
Object {
  "wrapper": <SignupPage
    signIn={[Function]}
    signUp={[Function]}
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  />,
}
`;
